src/codegen/mod.rs

```diff
@@ -3708,13 +3708,11 @@ impl CodeGenerator for ObjCInterface {
         debug_assert!(item.is_enabled_for_codegen(ctx));
 
         let mut impl_items = vec![];
-        let mut trait_items = vec![];
 
         for method in self.methods() {
-            let (impl_item, trait_item) =
+            let (impl_item, _trait_item) =
```

Comment:
if `trait_item` is always unused we should just remove it from the return value.